ui/apps/platform/src/Containers/NetworkGraph/common/NetworkGraphIcons.tsx

```diff
@@ -15,7 +15,7 @@ export const externalEntitiesBadgeColor = '#000000';
 
 export function DeploymentIcon(props) {
     return (
-        <Badge {...props} style={{ backgroundColor: deploymentBadgeColor }}>
+        <Badge {...props} style={{ backgroundColor: deploymentBadgeColor, width: 15 }}>
```

Comment:
nit: `15` may work, but I think only some properties like `line-height` and `opacity` are allowed to be unitless by CSSLint. (can't find the reference right now)
```suggestion
        <Badge {...props} style={{ backgroundColor: deploymentBadgeColor, width: '15px' }}>
```

Reply:
Oops, I think I forgot to remove this. I don't think the width changed whether I did a number or string with a px